src/components/common/table/table.js

```diff
@@ -119,15 +179,84 @@ class Table extends Component{
                         <i onClick={this.props.togleAddEmployeeModal} className="fa fa-user-plus"></i>
                     </div>
                 </div>
+                }
+                
+                {this.state.currentOpenedRowId !== null && 
+                
+                <Modal
+                    key={1}
+                    open={this.state.opinionModal}
+                    classNames={{ modal: "Modal Modal-add-owner" }}
+                    contentLabel="Employee opinion modal"
+                    onClose={() => this.setState({opinionModal: !this.state.opinionModal})}>
```

Comment:
In this case, it might not be a problem, but you must consider whether the state is changed. If this function would be called twice in a row, very fast, the change would occur not properly.